daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Transaction.scala

```diff
@@ -472,9 +470,9 @@ object GenTransaction extends value.CidContainer3[GenTransaction] {
 
 object Transaction {
 
-  @deprecated("use com.daml.lf.transaction.NodeId", since = "1.3.0")
+  @deprecated("use com.daml.lf.transaction.NodeId", since = "1.4.0")
```

Comment:
So, which is it? 1.3.0 or 1.4.0?

Reply:
When I wrote my previous PR (#6714), I though it will go in the release of yesterday (1.3.0).
I did not, so I take advantage of this PR, to fix the mistake.